src/views/Dashboard/Dashboard.tsx

```diff
@@ -26,7 +20,7 @@ export const Dashboard = () => {
         </Column>
         <Column percentageWidth={lgMatch ? 30 : 100}>
           <AccountSummary />
-          <TransferForm />
+          <TransferForm showContactsButton={pikeEnabled} />
```

Comment:
No I cannot. Because in `/contacts` page, even though the `pike-enabled` flag is set, it's no point to show the contacts button. 

Other words - the `TransferForm` is used twice - that's why I need this prop:
![image](https://github.com/bitcoin-sv/spv-wallet-web-frontend/assets/152964795/bf076e37-0b31-409d-a1d4-d2132f9fbae8)

Reply:
ShowContactsButton uses only in PaymailInput. TransferForm only is a proxy layer.

So it can be used like this
```suggestion
          export const PaymailInput = forwardRef<HTMLInputElement, PaymailInputProps>(({ showContactsButton, ...props }, ref) => {
  const paymailDomain = usePaymailDomain()
  const {pike_enabled} = useServerConfig()

  return (
    <Input ref={ref} {...props} type="text" labelText={`Paymail (example@${paymailDomain})`}>
      {pike_enabled && (
        <InputLinkButton to="/contacts">
          <ContactsIcon />
        </InputLinkButton>
      )}
    </Input>
  )
})
```

Reply:
No, because in your sulution the `ContactsIcon` would show in the `/contacts` subpage - and I wanted not to show it there. I think I'll just explain it to you on p2p meeting.